### ISSUE TITLE
[Adaptive Cards- Action.showcard] :  'What do you think' label is not associated with its button control.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9027]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: Dev (OS Build 27695.1000)
Browser Version: Version 130.0.2849.1 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to the 'Schema Explorer' option and press enter key to activate it.
3. Press tab key and navigate to the 'Action.ShowCard' link and press enter key to activate it.
4. Press tab key and navigate to the 'Neat' button and observe the issue.

**Actual Result:**
While navigating to the 'Neat' button, 'What do you think?' label is not associated with its button control. While navigating on the Neat button screen reader is announcing only as **'Neat button.'**

**Expected Result:**
'What do you think?' label should be associated with button control so that when the screen reader focus lands on it then screen reader should announce the information as **'What do you think? Neat button.'**

**Note:**
1. Same issue is repro with NVDA.
2. Same issue is repro under Input.Text and column set screen.
3. Same issue is repro throughout the application.

**User Impact:**
It will impact the screen reader user as label is not associated with its controls . User will be confuse about the control like this button related to which area.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![What do you think' label is not associated with its button control](https://github.com/user-attachments/assets/01b47062-5504-44bb-90dd-610ff09705d4)

https://github.com/user-attachments/assets/22e62bf7-c1bd-4e18-b29e-3df444cf8bf7

### Card JSON

```JSON
NA
```
